src/index.tsx

```diff
@@ -4,16 +4,16 @@ import './index.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import configureStore from './configureStore';
-import {Provider} from 'react-redux';
-import {AppInitializer} from './logic/initializer/AppInitializer';
+import { Provider } from 'react-redux';
```

Comment:
[minor]
Please don't change the code format without reason. This will make it difficult to review the code.

Reply:
yeah I will change it imedialy